akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -177,7 +177,12 @@ object ShardCoordinator {
   private val emptyRebalanceResult = Future.successful(Set.empty[ShardId])
 
   /**
-   * The default implementation of [[ShardCoordinator.LeastShardAllocationStrategy]]
+   * Deprecated: Use [[akka.cluster.sharding.LeastShardAllocationStrategy]] instead.
+   * The new rebalance algorithm was included in Akka 2.6.10. It can reach optimal balance in
+   * less rebalance rounds (typically 1 or 2 rounds). The amount of shards to rebalance in each
+   * round can still be limited to make it progress slower.
+   *
+   * This implementation of [[ShardCoordinator.ShardAllocationStrategy]]
```

Comment:
Isn't it a bit too eager to deprecate right away, I'd rather see a period of experimental before we deprecate the old one.

Reply:
We could wait with deprecation.

Reply:
not deprecated yet: 230f9467d9